agave/repositories/mongodb_repository.py

```diff
@@ -0,0 +1,54 @@
+from typing import Callable
+
+from cuenca_validations.types import QueryParams
+from mongoengine import Document
+from mongoengine import DoesNotExist as DoesNotExist
+
+from ..exc import ModelDoesNotExist
+from .base_repository import BaseRepository
```

Comment:
use `model` or `collection` instead of `repository`

Reply:
`collection` it's ok